tests/testthat/tests_modelEstimation.R

```diff
@@ -0,0 +1,140 @@
+context("testing modelEstimation")
+
+test_that("modelEstimation defaults works", {
+  skip_on_cran()
+  
+  eList_MO <- Missouri_eList
+  info_orig <- getInfo(eList_MO)
+  daily_orig <- getDaily(eList_MO)
+  sample_orig <- getSample(eList_MO)
+  surfaces_orig <- getSurfaces(eList_MO)
+  
+  eList_MO_modeled <- modelEstimation(eList_MO)
+  info <- getInfo(eList_MO_modeled)
+  daily <- getDaily(eList_MO_modeled)
+  sample <- getSample(eList_MO_modeled)
+  surfaces <- getSurfaces(eList_MO_modeled)
+  
+  ## INFO ##
+  
+  # info retained columns are unaltered after running modelEstimation
```

Comment:
None are removed, some are added and some are just translated from the original to the modeled result. Here, I'm testing that the columns that are translated aren't altered.